Cargo.toml

```diff
@@ -129,7 +129,7 @@ etcd-client = "0.14"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "a7274ddce299f33d23dbe8af5bbe6219f07c559a" }
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "2d52b660fa13ed3fb6d64d64d8dd6bfaf4ca3ef6" }
```

Comment:
Can you link the pr of the proto?

Reply:
https://github.com/GreptimeTeam/greptime-proto/pull/224